mix.exs

```diff
@@ -1,14 +1,25 @@
 defmodule ExCorreios.MixProject do
   use Mix.Project
 
+  @version "1.1.3"
+  @repo_url "https://github.com/locaweb/ex-correios"
+
   def project do
     [
       app: :ex_correios,
-      version: "1.1.3",
+      deps: deps(),
       elixir: "~> 1.10",
       elixirc_paths: elixirc_paths(Mix.env()),
       start_permanent: Mix.env() == :prod,
-      deps: deps()
+      version: @version,
+
+      # Hex
+      description: "",
```

Comment:
Maybe we could use the README description as the base for this? Something like

Elixir client that integrates with Brazilian Correios API.

Reply:
Great!!! :) 
Fixed!!!